test/clearingHouse/ClearingHouse.stopMarket.test.ts

```diff
@@ -222,11 +223,13 @@ describe("Clearinghouse StopMarket", async () => {
                 mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
                     return [0, parseUnits("200", 6), 0, 0, 0]
                 })
+                await accountBalance.mockMarkPrice(baseToken.address, parseEther("200"))
 
                 // set a lower price to have loss on baseToken2 market
-                mockedBaseAggregator2.smocked.latestRoundData.will.return.with(async () => {
+                mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
```

Comment:
```suggestion
                mockedBaseAggregator2.smocked.latestRoundData.will.return.with(async () => {
```

Reply:
Fixed in 7b8a7a3ea400702a74b58f1594349c410ca28a00